backend/.env.example

```
@@ -8,6 +8,7 @@ POSTGRES_DB=
 SECRET_KEY=your-secret-key-change-this-in-production
 ALGORITHM=HS256
 ACCESS_TOKEN_EXPIRE_DAYS=30
+SECURE_COOKIES=False    # TODO: set to True in prod
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)